modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -86,7 +86,23 @@ import {
   BubbleMenu,
   FloatingMenu
 } from '@tiptap/vue-2';
-import StarterKit from '@tiptap/starter-kit';
+// Starter Kit extensions
```

Comment:
I was going to say this belongs in a separate PR, but ✅ because pnpm is forcing us to address it now.